Objects/listobject.c

```diff
@@ -2659,38 +2659,56 @@ list.index
     start: slice_index(accept={int}) = 0
     stop: slice_index(accept={int}, c_default="PY_SSIZE_T_MAX") = sys.maxsize
     /
+    *
+    key: object=NULL
```

Comment:
AC has problems with `NULL` default. To check this please run `inspect.signature(list.index)` before and after.

Reply:
are you saying that `object=NULL` is nowhere used in the code despite being a documented convenience way of signifying optional `PyObject` values?  So, either fix AC, or the docs?
I was an active contributor before AC was introduced and I'm only learning it now.
I guess you want me to turn this into `object=None` and make comparisons with `PyNone` in the code.

Reply:
There is a bunch of these in the code:  https://github.com/search?q=repo%3Apython%2Fcpython+%22+object+%3D+null%22&type=code